jsonschema/cli.py

```diff
@@ -66,21 +66,32 @@ def validation_success(self, **kwargs):
         self._stdout.write(self._formatter.validation_success(**kwargs))
 
 
+def _json_formatter(x):
+    return json.dumps(x, indent=4, sort_keys=True)
+
+
 @attr.s
 class _PrettyFormatter(object):
+    _MESSAGE_BAR_CHAR = "═"
+    _MESSAGE_CORNER_CHARS = "╒", "╕"
+    _MESSAGE_FORMAT = "{}══[{}]═══({})"
+    _MESSAGE_MAX_LENGTH = 79
 
-    _ERROR_MSG = dedent(
-        """\
-        ===[{type}]===({path})===
+    def _message_line(self, path, type, header=False):
+        begin_char, end_char = self._MESSAGE_CORNER_CHARS if header \
```

Comment:
Minor style: jsonschema follows PEP8 / generally avoids using backslashes for continuation. Can you switch these to use parentheses instead?

Though, I think this will be a lot clearer if we split it into `_header_line` and `_non_header_line` -- can you give that a shot? It should remove a bunch of the conditional behavior here.

Reply:
I usually don't like backslashes, either.  I got in the habit of using it because my coworkers do like them.  I'll change this code.